safe_app/src/tests/append_only_data.rs

```diff
@@ -350,6 +350,7 @@ fn managing_permissions_for_an_app() {
 // The client then revokes the app by removing it from its list of authorised apps. The app should not
 // be able to append to the data anymore. But it should still be able to read the data since if it is published.
 // The client tries to delete the data. It should pass if the data is unpublished. Deleting published data should fail.
+#[ignore]
```

Comment:
to remove, or?

Reply:
This test fails for an unknown reason(mpsc channels error out when sending and receiving, not sure why). This is a chunky test though, thought of addressing this at the end. Hence the `ignore`.

Reply:
I'm okay with that if we open an issue to address this in the milestone 👍